examples/summarization/bart/finetune.py

```diff
@@ -152,6 +152,11 @@ def add_model_specific_args(parser, root_dir):
             required=True,
             help="The input data dir. Should contain the dataset files for the CNN/DM summarization task.",
         )
+        parser.add_argument(
```

Comment:
I think this should go in `lightning_base.py` and be called `tb_logging_dir`. Otherwise, LGTM.